Ports/ruby/package.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+source version.sh
+
+port=ruby
+version="${RUBY_VERSION}"
+workdir="ruby-${version}"
```

Comment:
This is implied, please don't specify it

Reply:
I can take workdir out, but ./package.sh fails if I remove port or version.